dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/README.md

```diff
@@ -240,6 +326,15 @@ public async Task MyMethodAsync(int a, int b, int c, int d, CancellationToken ca
 }
 ```
 
+This does however exclude `this` params that are used as part of extension methods:
+```csharp
+// No parameter count violation.
+public static string ExampleExtension(this Person person, int a, int b, int c, int d, int e);
```

Comment:
awesome 🎉 

I _think_ the default parameter max is `4` though (might be wrong there), so I think this should go up to `d` and the below go up to `e` if that makes sense?